analyses/tcga-capture-kit-investigation/scripts/CrossMap.sh

```diff
@@ -0,0 +1,22 @@
+# This script downloads the chain file from UCSC, converts the  BED files from hg19 to Gh38 ans sorts and mereges (in case there are  any overlaps in the BED regions)
+
+
+## Downloading chain.gz file for CrossMap command 
+wget http://hgdownload.soe.ucsc.edu/goldenPath/hg19/liftOver/hg19ToHg38.over.chain.gz  -O ../../scratch/hg19ToHg38.over.chain.gz
+
+## This command takes every BED files downloaded and uses CrossMap tool to convert hg19 coordinates to Gh38
+for i in `ls ../../scratch/*.targetIntervals.bed`; do 
+  out=$(echo $i | sed 's/.bed/.Gh38.bed/g')
+  CrossMap.py bed ../../scratch/hg19ToHg38.over.chain.gz $i $out
+done
+
+
+## This command uses sort and merge from bedtools to merge any overalpping BED regions
+for i in `ls ../../scratch/*.targetIntervals.Gh38.bed`; do 
+  bedtools sort -i $i \
+  | bedtools merge \
+  > results/$(basename $i)
```

Comment:
This will also presumably require the addition of `mkdir -p results/bedfiles/` somewhere in this script.
```suggestion
  > results/bedfiles/$(basename $i)
```

Reply:
Please review https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/597/commits/4e54fc3421bb9474b8de73f485c4a1dfb17e3f59

Reply:
Looks good! Should be merged when CI finishes.